resources/eventing/charts/nats-controller/values.yaml

```diff
@@ -10,7 +10,7 @@ image:
   # name is the name of the container image for the nats-eventing-controller
   name: "eventing-controller-nats"
   # tag is the container tag of the eventing-controller image
-  tag: "e2185713"
+  tag: "PR-11010"
```

Comment:
Please use the final image SHA instead of the PR image.

Also, please update the image here as well `resources/eventing/charts/controller`.

Reply:
Done